metron-platform/metron-common/README.md

```diff
@@ -87,6 +87,7 @@ but a convenient index is provided here:
 | [`es.port`](../metron-elasticsearch#esport)                                                                           | Indexing      | String     | N/A                                     |
 | [`es.date.format`](../metron-elasticsearch#esdateformat)                                                              | Indexing      | String     | `es_date_format`                        |
 | [`es.client.settings`](../metron-elasticsearch#esclientsettings)                                                      | Indexing      | Object     | N/A                                     |
+| [`indexing.writer.metronId`](../metron-indexing#elasticsearch)                                                        | Indexing      | Boolean    | N/A                                     |
```

Comment:
Maybe `indexing.writer.elasticsearch.metronId` to make is more clear that this setting only applies to Elasticsearch, not Solr?

Or `indexing.writer.elasticsearch.setDocumentId` if you agree with my other comments.

Reply:
Done.